core/src/main/java/org/geysermc/geyser/network/UpstreamPacketHandler.java

```diff
@@ -173,11 +171,6 @@ public PacketSignal handle(LoginPacket loginPacket) {
             return PacketSignal.HANDLED;
         }
 
-        // Hack for... whatever this is
-        if (loginPacket.getProtocolVersion() == Bedrock_v567.CODEC.getProtocolVersion() && !session.getClientData().getGameVersion().equals("1.19.60")) {
-            session.getUpstream().getSession().setCodec(Bedrock_v568.CODEC);
-        }
-
```

Comment:
Double check this please

Reply:
It's for some weirdness with the 1.19.62 and 1.19.63 version; we can drop it

Reply:
Correct.